src/colourmap.cpp

```diff
@@ -98,12 +98,12 @@ namespace MR
               }),
 
           Entry ("PET",
-              "color.r = 2.0*amplitude - 0.5;\n"
+              "color.r = clamp (2.0*amplitude - 0.5, 0.0, 1.0);\n"
               "color.g = clamp (2.0 * (0.25 - abs (amplitude - 0.25)), 0.0, 1.0) + clamp (2.0*amplitude - 1.0, 0.0, 1.0);\n"
               "color.b = 1.0 - (clamp (1.0 - 2.0 * amplitude, 0.0, 1.0) + clamp (1.0 - 4.0 * abs (amplitude - 0.75), 0.0, 1.0));\n",
               [] (float amplitude) { return Eigen::Array3f (clamp (2.0f * amplitude - 0.5f),
-                                                            clamp (0.25f - abs (amplitude - 0.25f)) + clamp (2.0f * (amplitude - 0.5)),
-                                                            clamp (1.0f - 2.0f * amplitude) + clamp (1.0 - 4.0 * abs (amplitude - 0.75))); }),
+                                                            clamp (2.0f * (0.25f - abs (amplitude - 0.25f))) + clamp (2.0f * amplitude - 1.0f),
+                                                            1.0f - (clamp (1.0f - 2.0f * amplitude) + clamp (1.0f - 4.0f * abs (amplitude - 0.75f)))); }),
```

Comment:
If I did indeed make a meal of this, I'd like to cherry-pick and include in `3.0.1`.

Reply:
There is definitely an issue in 3.0.0. I was in two minds about creating a separate issue for this one; maybe I should have.. Happy for it to be pushed to 3.0.1.